src/GuzzleBundleCachePlugin.php

```diff
@@ -92,4 +74,23 @@ public function getPluginName(): string
     {
         return 'cache';
     }
+
+    /**
+     * @param ContainerBuilder $container
+     *
+     * @return Definition
+     */
+    protected function getInvalidateRequestSubscriberDefinition(ContainerBuilder $container) : Definition
+    {
+        $invalidateRequestSubscriberName = 'guzzle_bundle_cache_plugin.event_subscriber.invalidate_request';
+
+        if (!$container->hasDefinition($invalidateRequestSubscriberName)) {
+            $invalidateRequestSubscriberDefinition = new Definition(InvalidateRequestSubscriber::class);
```

Comment:
no need to create it dynamically now. We can declare it in the `.xml` file.

Reply:
moved